storybook/stories/x-axis/data-object.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
what's the difference between this and the above file?

Reply:
I renamed the file from `data-object-styles.js` to `data-object.js`, and it seems I didn't commit the deletion of the former.